src/app/components/booking/index.tsx

```diff
@@ -53,6 +55,11 @@ export default function BookingComponent({
     setOpenPrepaymentModal(false);
   };
 
+  const page = async (file: any) => {
+    const data = await uploadFile(file)
+    return data
+  }
+
```

Comment:
This function which name is horrible `page`  😆  is not really needed
We can just call to the API directly in the submit  form

`await uploadFile(formData["ID_Front_Picture"])`
`await uploadFile(formData["ID_Back_Picture"])`